pkg/controllers/node/node_syncer.go

```diff
@@ -37,6 +38,19 @@ func (c *NodeController) initSyncer() {
 
 func (c *NodeController) OnStatusUpdated(status bapi.SyncStatus) {
 	logrus.Infof("Node controller syncer status updated: %s", status)
+	c.syncStatus = status
+
+	switch status {
+	case bapi.ResyncInProgress:
+		// Starting the resync so clear our node cache.
+		c.nodeCache = make(map[string]*api.Node)
+
+	case bapi.InSync:
+		err := c.syncAllAutoHostendpoints()
+		if err != nil {
+			logrus.WithError(err).Fatal()
```

Comment:
missing a log message here  - `.Fatal("failed to sync host endpoints on startup")` maybe?

Reply:
7da10da